enrollment-server/src/test/java/com/wultra/app/docverify/zenid/provider/ZenidDocumentVerificationProviderTest.java

```diff
@@ -47,7 +52,7 @@
  * @author Lukas Lukovsky, lukas.lukovsky@wultra.com
  */
 @SpringBootTest(classes = EnrollmentServerTestApplication.class)
-@ActiveProfiles("external-service")
+@ActiveProfiles({"external-service"})
```

Comment:
Is this change required?

Reply:
aah, thanks
I had two profiles there and removed the other, no need for an array any more